driver-core/src/main/com/mongodb/MongoClientSettings.java

```diff
@@ -184,12 +184,10 @@ public DnsClient getDnsClient() {
     }
 
     /**
-     * Gets the {@link InetAddressResolver} to use for looking up the {@link java.net.InetAddress} instances for each host.
-     *
-     * <p>If set, it will be used to look up the {@link java.net.InetAddress} for each host, via
-     * {@link InetAddressResolver#lookupByName(String)}. Otherwise, {@link java.net.InetAddress#getAllByName(String)} will be used.
+     * Gets the explicitly set {@link InetAddressResolver} to use for looking up the {@link java.net.InetAddress} instances for each host.
```

Comment:
Just FYI, we generally don't update Javadoc in patch releases, because we don't republish the Javadoc for patch releases.  But it doesn't do any harm, so it's ok to leave this.

Reply:
Right, thanks! It's still better to have the source code consistent with the published API documentation, so I rolled back the change.